### PR TITLE
Add private q's to RALPMH-ILD project

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add private q's (`K-Build` questionnaire) to RALPMH-ILD project and remove `LIPF`

### CODE REVIEW

File: RALPMH-ILD-Lon-s1/protocol.json [hunk 1]
```diff
@@ -78,13 +78,13 @@
...
-      "showIntroduction": "always",
+      "name": "K_Build_Private",
+      "showIntroduction": false,
       "showInCalendar": true,
       "order": 0,
       "questionnaire": {
-        "repository": "https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/",
-        "name": "lipf",
+        "repository": "https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions-Private/master/questionnaires/",
+        "name": "k_build_private",
```
Comment (yatharthranjan) [inline RALPMH-ILD-Lon-s1/protocol.json:87]: This looks good to me. Although a couple of points-

- We need to upgrade the production appserver to latest, release the app to production, and make sure enough people are updated to the new app version before merging this.
- I just noticed the questions in K-Build are based on a 2 week period, so delivering them daily is an oxymoron. I am just confirming with the clinical teams on the ways to proceed here, either the protocol will be 14 days or the questions will need to be modified.

File: RALPMH-ILD-Lon-s1/protocol.json [hunk 2]
```diff
@@ -78,13 +78,13 @@
...
         "avsc": "questionnaire"
       },
       "startText": {
```
Comment (yatharthranjan) [inline RALPMH-ILD-Lon-s1/protocol.json:90]: you can change this to "over the last 2 weeks" and `showIntroduction` to true.

File: RALPMH-ILD-Lon-s1/protocol.json (hunk 1 above)
Comment (mpgxvii) [reply RALPMH-ILD-Lon-s1/protocol.json:87]: > We need to upgrade the production appserver to latest, release the app to production, and make sure enough people are updated to the new app version before merging this.

Yup will do that.

**Review by yatharthranjan — APPROVED** [review]
LGTM, given this is true
> We need to upgrade the production appserver to latest, release the app to production, and make sure enough people are updated to the new app version before merging this.

**Pull request merged**